index.html

```diff
@@ -10,346 +10,305 @@
-                    <input type="email" placeholder="Email address" name="email" required/>
-                    <textarea maxlength="500" placeholder="Enter text here" rows="5" name="text" required></textarea>
-                    <button type="submit">Get in touch</button>
-                  </form>
-                </section>
-              </section>
-              <section>
-                <section class="icon">
-                  <ul>
-                    <li>
-                      <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                        ><i class="fa-brands fa-linkedin-in"></i
-                      ></a>
-                    </li>
-                    <li>
-                      <a href=""><i class="fa-brands fa-twitter"></i></a>
-                    </li>
-                    <li>
-                      <a href="https://github.com/dagmawiii03"
-                        ><i class="fa-brands fa-github"></i
-                      ></a>
-                    </li>
-                    <li>
-                      <a href=""><i class="fa-solid fa-envelope"></i></a>
-                    </li>
-                  </ul>
-                  <p class="grey">dagmawi.yoha03@gmail.com</p>
-                  <br />
-                  <br />
-                </section>
-              </section>
-            </section>
-          </footer> 
-        </main>
-</body>
-</html>
\ No newline at end of file
+          </article>
+        </li>
+      </ul>
+    </section>
+    <footer class="footer-container">
+      <section class="footer-form">
+        <h1 class="footer-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
+        <form action="https://formspree.io/f/xoqzdylg" method="post">
+          <ul class="form-fields">
+            <li class="input-name">
+              <input
+                type="text"
+                name="name"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li class="input-email-form">
+              <input
+                class="input-email"
+                type="email"
+                name="email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="text_area"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+            </li>
+            <li class="submit-button">
+              <button type="submit">Get in touch</button><br />
+              <small></small>
+            </li>
+          </ul>
+        </form>
+      </section>
+      <ul class="social-media">
```

Comment:
Kindly add the gray line before the social media icon.
<img width="853" alt="Screenshot 2023-02-25 at 11 57 55 PM" src="https://user-images.githubusercontent.com/25887253/221380236-34175537-3ce4-49d7-af02-01e7282f982b.png">